src/overlays/actors/ovl_Bg_Mizu_Movebg/z_bg_mizu_movebg.c

```diff
@@ -21,21 +38,357 @@ const ActorInit Bg_Mizu_Movebg_InitVars = {
     (ActorFunc)BgMizuMovebg_Update,
     (ActorFunc)BgMizuMovebg_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Mizu_Movebg/func_8089DC30.s")
 
+static f32 D_8089EB40[] = {
+    -115.200005,
+    -115.200005,
+    -115.200005,
+    0.0f,
+};
```

Comment:
id prob remove the comma on this, it looks short enough. unless you think its something modders would want to add more numbers to